wondrous-app/components/organization/pods/index.tsx

```diff
@@ -0,0 +1,227 @@
+import Link from 'next/link';
+import { useCallback, useEffect, useState } from 'react';
+import { useLazyQuery } from '@apollo/client';
+import { Grid, Typography } from '@mui/material';
+
+import { useMe } from 'components/Auth/withAuth';
+
+import { useOrgBoard } from 'utils/hooks';
+import { capitalize } from 'utils/common';
+import { ENTITIES_TYPES, PERMISSIONS } from 'utils/constants';
+import { GET_ORG_PODS, GET_USER_PODS } from 'graphql/queries';
+import { parseUserPermissionContext } from 'utils/helpers';
+
+import palette from 'theme/palette';
+import typography from 'theme/typography';
+
+import PlusIcon from 'components/Icons/plus';
+import { CreateEntity } from 'components/CreateEntity';
+import {
+  CreateNewPodButton,
+  CreateNewPodIconWrapper,
+  PodItemWrapper,
+  SearchPods,
+  StyledTab,
+  StyledTabs,
+} from './styles';
+
+import PodItem from './PodItem';
+import { PodView } from './constants';
+
+const TabLabel = ({ label, count, isActive }) => (
+  <Grid sx={{ display: 'flex', alignItems: 'center', gap: '10px' }}>
+    <Typography
+      sx={{
+        color: isActive ? palette.white : palette.grey51,
+        fontWeight: 500,
+        fontFamily: typography.fontFamily,
+        fontSize: '14px',
+      }}
+    >
+      {label}
+    </Typography>
+    {!!count && (
+      <Typography
+        sx={{
+          color: isActive ? palette.white : palette.grey51,
+          background: isActive ? palette.grey87 : palette.grey87,
+          padding: '2px',
+          borderRadius: '4px',
+          fontWeight: 500,
+          fontFamily: typography.fontFamily,
+          fontSize: '14px',
+        }}
+      >
+        {count}
+      </Typography>
+    )}
+  </Grid>
+);
+
+const Pods = (props) => {
+  const { orgData } = props;
+
+  const [activePodView, setActivePodView] = useState(PodView.ALL);
+  const [activePodsList, setActivePodsList] = useState([]);
+  const [showCreatePodModal, setShowCreatePodModal] = useState(false);
+
+  const user = useMe();
+  const { userPermissionsContext } = useOrgBoard() || {};
+  const permissions = parseUserPermissionContext({
+    userPermissionsContext,
+    orgId: orgData?.id,
+  });
+  const [getOrgPods, { data: orgPodsData }] = useLazyQuery(GET_ORG_PODS, {
+    fetchPolicy: 'cache-and-network',
+  });
+  const [getUserPods, { data: userPodsData }] = useLazyQuery(GET_USER_PODS, {
+    fetchPolicy: 'network-only',
+  });
+
+  const orgName = orgData?.name || orgData?.username;
+  const orgPods = orgPodsData?.getOrgPods;
+  const userPods = userPodsData?.getUserPods;
+  const orgPodsUserIsIn = userPods?.filter((pod) => pod.org?.id === orgData?.id);
+  const orgPodsUserIsNotIn = orgPods?.filter(
+    (pod) => !orgPodsUserIsIn?.find((podUserIsIn) => podUserIsIn.id === pod.id)
+  );
+  const canUserCreatePods =
+    permissions.includes(PERMISSIONS.FULL_ACCESS) || permissions.includes(PERMISSIONS.MANAGE_POD);
+
+  useEffect(() => {
```

Comment:
we can replace useLazyQuery with useQuery for this scenario and use the skip flag, this way we don't need an additional hook ( useEffect )

`const {data: userPodsData} = useQuery(GET_USER_PODS, {fetchPolicy: 'network-only', skip: !orgData?.id, variables: {...}})`  I believe the same applies for getUserPods